mail/mu/Portfile

```diff
@@ -4,39 +4,49 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           elisp 1.0
 
-github.setup        djcb mu 1.0 v
-license             GPL-3
+name                mu
+platforms           darwin
 categories          mail
-maintainers         nomaintainer
+license             GPL-3
+maintainers         {@ra1nb0w irh.it:rainbow}
```

Comment:
Would it be ok to keep the port under openmaintainer policy? If so, please add the `openmaintainer` keyword at the end.

Reply:
I lost this, thank you